kong/plugins/response-ratelimiting/migrations/000_base_response_rate_limiting.lua

```diff
@@ -59,7 +59,7 @@ return {
 
             BEGIN
               INSERT INTO response_ratelimiting_metrics (identifier, period, period_date, service_id, route_id, value)
-                   VALUES (i, p, p_date, r_id, s_id, v);
+                   VALUES (i, p, p_date, s_id, r_id, v);
```

Comment:
Yes, that's the one — I did look at that other PR earlier today, but that PR is for the 001_14_to_15 migrations, so I think it would be fine to single out this bugfix on the 000_base migration here, maybe?

Reply:
yep yep